ChangeLog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## Version 1.58.0
+* Libs/Rust: Nullable fields on now correctly `Option<T>` instead of `T`, this affects `ListResponseXXXOut.iterator` and `EnvironmentOut.settings`
```

Comment:
```suggestion
* Libs/Rust: fix regression in previous version. Nullable fields on now correctly `Option<T>` instead of `T`, this affects `ListResponseXXXOut.iterator` and `EnvironmentOut.settings`
```